project-base/app/assets/js/admin/validation/parameterNameValidator.js

```diff
@@ -0,0 +1,14 @@
+import 'framework/common/components';
+import Register from 'framework/common/utils/Register';
+
+function parameterNameValidator ($container) {
```

Comment:
shouldn't it be in the framework package?

Reply:
sure it should, thank you, fixed in 338ebb0394